docs/using-radicle/tracking-and-viewing.md

```diff
@@ -17,22 +17,26 @@ in the peer selector on your project page. From there, you can add their Device
 ID. This is also where you can switch between the different views of peers to
 see their changes before fetching them locally.
 
+<!-- TODO: REPLACE WITH PATCHES SCREENSHOT -->
+
 <video style="border-radius: 16px; margin: 16px 0; width: 100%; height: 100%;"
   playsinline autoplay muted loop>
   <source src="/img/add-remote.mp4" type="video/mp4">
 </video>
 
 Adding a peer as a remote automatically tracks their view of your project,
-subscribing you to any updates they push to their branches. It's important to
-note that these changes may not instantly appear. We rely on the connectedness
-of the network to ensure you hear about the changes from a peer you track. When
-patches and code review are introduced, it will be easier to keep track of
-peer's changes to your project. Until then, you'll have to manage remotes
-manually. 
+subscribing you to any updates they push to their branches as well as any
+patches they might have. It's important to note that these changes may not
+instantly appear. We rely on the connectedness of the network to ensure you hear
+about the changes from a peer you track. Eventually we want to introduce
+automatic subscribing to any patches that might be available by others to your
```

Comment:
I think we could remove the last two sentences here. This is docs on how it works, not a roadmap. What if we never do that, this sentence will haunt us.